### PR TITLE
Update/dbt workshops

### PR DESCRIPTION
**Pull request by amychen1776**
Updating the naming of the dbt workshops to fit our brand guidelines as well as correcting the workshop duration of the dbt Cloud workshop. 

Recommendation: The commits will be much clearer to see my changes

### CODE REVIEW

File: site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md [hunk 1]
```diff
@@ -0,0 +1,1447 @@
+<<<<<<< HEAD:site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md
```
Comment (jdanielmyers) [inline site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md:1]: Thank you for the PR! 

This file is broken and does not build. These merge conflicts need to be resolved before merging.

File: site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md [hunk 2]
```diff
@@ -0,0 +1,1447 @@
...
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Engineering, dbt, Data Sharing
+authors: Amy Chen, Dmytro Yaroshenko
+
+<<<<<<< HEAD:site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md
+# Accelerating Data Teams with dbt Core & Snowflake
+=======
+# Accelerating Data Teams with dbt & Snowflake
+>>>>>>> dc95377a69000661e445a9e05d1a5d9a4c3bbd1c:site/sfguides/src/data_engineering_with_dbt/data_engineering_with_dbt.md
```
Comment (jdanielmyers) [inline site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md:19]: merge conflicts breaking the build. Also, the image references seem to be breaking when trying to build.

File: package-lock.json [hunk 3]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "codelab-elements",
-  "version": "4.0.0",
+  "version": "1.0.1",
```
Comment (jdanielmyers) [inline package-lock.json:3]: what is this change for? If it is not needed please remove it from PR.

**Comment by jdanielmyers** [comment]
Thank you for the PR! 

It looks like the PR will be changing the URLs of these Quickstarts, please add a URL redirect from the old URLs to the new ones using the `firebase.json` file:  https://github.com/amychen1776/sfquickstarts/blob/59e4dae5e113b72f3decfe3c5b64900f0f091a68/site/firebase.json#L46

File: package-lock.json (hunk 3 above)
Comment (amychen1776) [reply package-lock.json:3]: Thanks for catching that; it must have sneaked in when I pulled from the main branch

**Comment by amychen1776** [comment]
@jdanielmyers Looking at the firebase.json, could you tell me what the URLs are pulling from? On that file, there's only one dbt object. 

Thank you so much for your help here!

**Comment by amychen1776** [comment]
Closing this because I think I finally fixed my fork so have a new PR

**Pull request closed without merging**